Source: http://bzr.linuxfoundation.org/loggerhead/openprinting/cups-filters/revision/7332
Upstream: Fixed in bzr
Reason: Fix build with Poppler 0.31.0

=== modified file 'NEWS'
--- NEWS	2015-02-14 00:35:53 +0000
+++ NEWS	2015-02-18 03:17:28 +0000
@@ -1,6 +1,12 @@
 NEWS - OpenPrinting CUPS Filters v1.0.65 - 2015-02-13
 -----------------------------------------------------
 
+CHANGES IN V1.0.66
+
+	- pdftoopvp: Added conditionals to also build with Poppler 0.31.0
+	  and newer. Thanks to Armin K. (krejzi at email dot com) for the
+	  patch (Bug #1254).
+
 CHANGES IN V1.0.65
 
 	- cups-browsed: Listen for NetworkManager changes (Red Hat bug #975933).

=== modified file 'filter/pdftoijs.cxx'
--- filter/pdftoijs.cxx	2013-08-02 13:52:04 +0000
+++ filter/pdftoijs.cxx	2015-02-18 03:17:28 +0000
@@ -436,7 +436,11 @@
   }
 
   out = new SplashOutputDev(cmode,rowpad/* row padding */,
-    reverseVideo,paperColor,gTrue,gFalse);
+    reverseVideo,paperColor,gTrue
+#if POPPLER_VERSION_MAJOR == 0 && POPPLER_VERSION_MINOR <= 30
+    ,gFalse
+#endif
+    );
 #if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 19
   out->startDoc(doc);
 #else

=== modified file 'filter/pdftoopvp/OPVPOutputDev.cxx'
--- filter/pdftoopvp/OPVPOutputDev.cxx	2014-03-10 11:24:29 +0000
+++ filter/pdftoopvp/OPVPOutputDev.cxx	2015-02-18 03:17:28 +0000
@@ -278,7 +278,11 @@
 				    gFalse,
                                     gFalse,
 #endif
+#if POPPLER_VERSION_MAJOR == 0 && POPPLER_VERSION_MINOR <= 30
 				    globalParams->getAntialias());
+#else
+                                    gFalse);
+#endif
   for (i = 0; i < nT3Fonts; ++i) {
     delete t3FontCache[i];
   }

=== modified file 'filter/pdftoopvp/pdftoopvp.cxx'
--- filter/pdftoopvp/pdftoopvp.cxx	2013-08-02 14:46:20 +0000
+++ filter/pdftoopvp/pdftoopvp.cxx	2015-02-18 03:17:28 +0000
@@ -614,7 +614,9 @@
       ok = gFalse;
     }
   }
+#if POPPLER_VERSION_MAJOR == 0 && POPPLER_VERSION_MINOR <= 30
   globalParams->setAntialias("no");
+#endif
   if (quiet) {
     globalParams->setErrQuiet(quiet);
   }

=== modified file 'filter/pdftoraster.cxx'
--- filter/pdftoraster.cxx	2014-09-26 18:46:49 +0000
+++ filter/pdftoraster.cxx	2015-02-18 03:17:28 +0000
@@ -2100,7 +2100,11 @@
   }
 
   out = new SplashOutputDev(cmode,rowpad/* row padding */,
-    gFalse,paperColor,gTrue,gFalse);
+    gFalse,paperColor,gTrue
+#if POPPLER_VERSION_MAJOR == 0 && POPPLER_VERSION_MINOR <= 30
+    ,gFalse
+#endif
+    );
 #if POPPLER_VERSION_MAJOR > 0 || POPPLER_VERSION_MINOR >= 19
   out->startDoc(doc);
 #else

